README.md

```diff
@@ -8,10 +8,19 @@ npm install
 
```

Comment:
Looks like the README has reverted to a more generic auto-generated version. Seems like it would better to keep the current one?